### PR TITLE
Replace header dropdown with buttons

### PR DESCRIPTION
**Pull request by gaganyadav80**
Also decreased vertical divider indents to make it more visible and look better.

### CODE REVIEW

File: lib/widgets/toolbar.dart [hunk 1]
```diff
@@ -370,8 +370,9 @@ Widget defaultToggleStyleButtonBuilder(
 
 class SelectHeaderStyleButton extends StatefulWidget {
   final QuillController controller;
+  final double headerfontSize;
 
-  const SelectHeaderStyleButton({required this.controller, Key? key})
+  const SelectHeaderStyleButton({required this.controller, this.headerfontSize = 18.0, Key? key})
```
Comment (singerdmx) [inline lib/widgets/toolbar.dart:375]: headerFontSize

File: lib/widgets/toolbar.dart [hunk 2]
```diff
@@ -424,62 +425,46 @@ class _SelectHeaderStyleButtonState extends State<SelectHeaderStyleButton> {
...
-      ),
-      PopupMenuItem(
-        value: Attribute.h3,
-        height: iconSize * 1.77,
-        child: Text(_valueToText[Attribute.h3]!, style: style),
+  List<Attribute> _valueAttribute = [Attribute.header, Attribute.h1, Attribute.h2, Attribute.h3];
+  List<String> _valueString = ['N','H1','H2','H3'];
+
+  final theme = Theme.of(context);
+  final style = theme.textTheme.caption?.copyWith(fontWeight: FontWeight.bold, fontSize: (15.0/iconSize)*headerfontSize);
```
Comment (singerdmx) [inline lib/widgets/toolbar.dart:445]: please format

**Comment by singerdmx** [comment]
it seems `kIsWeb` is removed.
Does this change handle web?

**Comment by gaganyadav80** [comment]
> it seems `kIsWeb` is removed.
> Does this change handle web?

Just noticed that! I forgot about the web. I Will add it

**Comment by gaganyadav80** [comment]
@singerdmx  I think removing `kIsWeb` will not effect the outcome. Because I'm using `index` to get the values with separate Lists for Attributes and Strings.

So, these two piece of code will output the same thing wether `kIsWeb` is true or not.
```dart
final Map<Attribute, String> _valueToText = {
    Attribute.header: 'Normal text',
    Attribute.h1: 'Heading 1',
    Attribute.h2: 'Heading 2',
    Attribute.h3: 'Heading 3',
  };

child: Text(
            !kIsWeb
          ? _valueToText[value!]!
          : _valueToText[value!.key == 'header'
              ? Attribute.header
              : (value.key == 'h1')
                  ? Attribute.h1
                  : (value.key == 'h2')
                      ? Attribute.h2
                      : Attribute.h3]!,
         ),
```
AND
```dart
List<Attribute> _valueAttribute = [Attribute.header, Attribute.h1, Attribute.h2, Attribute.h3];
List<String> _valueString = ['N','H1','H2','H3'];

child: Text(_valueString[index]),
```

**Pull request merged**